riyaz/cli/__init__.py

```diff
@@ -58,19 +62,11 @@ def new():
 
 
 @contextlib.contextmanager
-def setup_db():
-    with tempfile.TemporaryDirectory(prefix="riyaz_") as tempdir:
-        config.database_path = os.path.join(tempdir, "riyaz.db")
-        migrate()
-
-        yield
-
-
-def reset_db():
-    drop_db()
+def setup_db(tempdir):
+    config.database_path = os.path.join(tempdir, "riyaz.db")
     migrate()
 
 
-def drop_db():
-    if os.path.exists(config.database_path):
-        os.unlink(config.database_path)
+def setup_assets(tempdir):
+    config.assets_path = os.path.join(tempdir, "assets")
+    os.makedirs(config.assets_path, exist_ok=True)
```

Comment:
Why do we need to make an assets directory in the tmp folder? Why can't you serve the assets from where they are?

Reply:
This directory is like the assets directory on a hosted server with multiple courses.

Users may want to keep their own assets in the local `assets/` organized differently.
If we load the assets from their paths into local `assets/` that would cause spam.

Reply:
Our URLs and paths would be like `/assets/instructor/1/kitten.png`, but that might not make sense for the course creator.